addons/binding/org.openhab.binding.nibeuplink/src/main/java/org/openhab/binding/nibeuplink/internal/connector/UplinkWebInterface.java

```diff
@@ -134,6 +136,10 @@ public void update(CommunicationStatus status) {
                             uplinkHandler.setStatusInfo(ThingStatus.OFFLINE, ThingStatusDetail.BRIDGE_OFFLINE,
                                     status.getMessage());
                             setAuthenticated(false);
+                        } else if (Code.FOUND.equals(status.getHttpCode())) {
```

Comment:
To be sure: `FOUND` or `NOT_FOUND`?

Minor improvement possible: Use a switch statement on `status.getHttpCode()`?

Reply:
Thanks for clarification.